core/domain/state_domain.py

```diff
@@ -943,6 +983,29 @@ def __init__(self, value, schema):
         self.value = value
         self.schema = schema
 
+    def _register_all_translatable_fields(self) -> None:
+        """Registration of all the fields for InteractionCustomizationArg."""
+
+        subtitled_htmls = self.get_subtitled_html()
+        for subtitled_html in subtitled_htmls:
+            html_string = subtitled_html.html
+            # Make sure we don't include content that only consists of
+            # numbers. See issue #13055.
+            if html_string != '' and not html_string.isnumeric():
```

Comment:
I think we are going to do this check inside the `_register_translatable_field`, right?

Reply:
True,
Empty string check:  removed from here.
html_string is not numeric: Is left unchanged.